packages/fxa-auth-server/.eslintrc.json

```diff
@@ -5,7 +5,13 @@
   "extends": ["plugin:fxa/server"],
   "plugins": ["fxa"],
   "parserOptions": {
-    "ecmaVersion": "2020",
+    "ecmaVersion": 2020,
```

Comment:
Changed this from `"2020"` to `2020` since VSCode was saying `"2020"` wasn't a valid/recommended value.